src/site/stages/build/options.js

```diff
@@ -236,4 +236,6 @@ async function getOptions(commandLineOptions) {
   return options;
 }
 
-module.exports = getOptions;
+module.exports = {
+  getOptions,
+};
```

Comment:
Was this change necessary for something?

Reply:
I found it necessary to get the `require`'s working properly.

Reply:
Did they not work properly before? This is the only change in this file.

Reply:
I'm able to build without the named export. Unless you've observed the `require`s not working properly somewhere else.

Reply:
It was needed for the preview server.